src/render/graphs/lineOverview.ts

```diff
@@ -1,108 +1,107 @@
-import { attribute, canvas, div, HtmlNode } from "@eeue56/coed";
-import {
-  Chart,
-  Colors,
-  Filler,
-  Legend,
-  LinearScale,
-  LineController,
-  LineElement,
-  PointElement,
-  Title,
-  Tooltip,
-} from "chart.js";
+import { attribute, class_, div, HtmlNode } from "@eeue56/coed";
 import { getDataPerPrompt } from ".";
 import { JournalEntry } from "../../types";
 import { dayToString, sortEntriesByDate } from "../../utils/dates";
 
-Chart.register(
-  Colors,
-  LinearScale,
-  LineController,
-  PointElement,
-  LineElement,
-  Tooltip,
-  Filler,
-  Title,
-  Legend
-);
+function createLineChartSvg(
```

Comment:
Convert this code to return a `HtmlNode` via the @eeue56/coed package. It has an `svg` export which you can use. Do not use innerHTML or similar.